examples/example-tls/README.md

```diff
@@ -43,14 +43,15 @@ You can use the following script to generate self-signed certificates for grpc-j
 mkdir -p /tmp/sslcert
 pushd /tmp/sslcert
 # Changes these CN's to match your hosts in your environment if needed.
+SERVER_CA_CN=localhost-ca # must be different than SERVER_CN
 SERVER_CN=localhost
 CLIENT_CN=localhost # Used when doing mutual TLS
 
 echo Generate CA key:
 openssl genrsa -passout pass:1111 -des3 -out ca.key 4096
 echo Generate CA certificate:
 # Generates ca.crt which is the trustCertCollectionFile
-openssl req -passin pass:1111 -new -x509 -days 365 -key ca.key -out ca.crt -subj "/CN=${SERVER_CN}"
+openssl req -passin pass:1111 -new -x509 -days 365 -key ca.key -out ca.crt -subj "/CN=${SERVER_CA_CN}"
 echo Generate server key:
 openssl genrsa -passout pass:1111 -des3 -out server.key 4096
 echo Generate server signing request:
```

Comment:
Lines 59 and 68 say "Self-signed..." which is incorrect, so we can fix that?

Reply:
what would you prefer?   I think self-signed is pretty clear, it means someone didnt give it to us.

Reply:
See https://en.wikipedia.org/wiki/Self-signed_certificate 

> In technical terms a self-signed certificate is one signed with its own private key.

Here server and client certs are signed by the ca cert's private key.